dist-material/release-docs/NOTICE

```diff
@@ -838,7 +838,43 @@ from and not be held liable to the user for any such damages as noted
 above as far as the program is concerned.
 
 ------
+===========================================================================
+nacos-1.3.0 Notice
+===========================================================================
+Nacos
+Copyright 2018-2019 The Apache Software Foundation
```

Comment:
Nacos is not belong to the ASF, the NOTICE is wrong.

Reply:
> The PR becomes messy and removes lots of commits from the upstream, I suggest to rebase or simply revert #4867 first and then upgrade the Nacos version immediately, the latter would be more friendly to reviewers

thanks for reviews